sky/spot/recovery_strategy.py

```diff
@@ -231,6 +231,8 @@ def _launch(self, max_retry=3, raise_on_failure=True) -> Optional[float]:
 
         The function will wait until the job starts running, but will leave the
         handling for the preemption to the caller.
+        The return value will only be None when the job fails to submit and the
```

Comment:
I think we should add this clarification to launch()'s docstr, i.e., updating the line `Returns: The job's submit timestamp, or None if failed.`. 

Here, this line repeats L242-244.